cfgov/cfgov/urls.py

```diff
@@ -37,6 +39,21 @@
 
 oah = SheerSite('owning-a-home')
 
+
+def flagged_wagtail_template_view(flag_name, template_name):
+    """View that serves Wagtail if a flag is set, and a template if not.
+
+    This uses the wagtail-sharing ServeView to ensure that sharing works
+    properly when viewing the page in Wagtail behind a flag.
+    """
+    return FlaggedTemplateView.as_view(
+        fallback=lambda request: ServeView.as_view()(request, request.path),
```

Comment:
I also wonder if we should just define this lambda (I know, E731) somewhere as `wagtail_serve_view` or something since we reuse it so often.

Reply:
I wondered about that too. It tends to be added and removed when flags are created and then deprecated.

Reply:
Yeah. If it's a variable with the lambda assigned then flake8 will complain. Maybe we should define it as a function in v1.views?